### ISSUE TITLE
a11y: TodayView voiceover pass

### ISSUE THREAD

**Issue by ualch9** [OneBusAway/OBAKit#430]

**Comment by ualch9** [comment]
Tested, and only one thing needed -- the time badge should have a voiceover text of "departing in 3 minutes" instead of "3 minutes".

![IMG_3615](https://user-images.githubusercontent.com/22162410/127536692-7aca80c0-abb9-46b7-bb85-9a4905be62ae.jpg)
